pkg/rpc/server/ospkg/server.go

```diff
@@ -39,7 +39,7 @@ func (s *Server) Detect(_ context.Context, req *proto.OSDetectRequest) (res *pro
 		return nil, err
 	}
 
-	s.vulnClient.FillInfo(vulns, false)
+	s.vulnClient.FillInfo(vulns, req.OsFamily, false)
```

Comment:
If we send a `ReportType` in a rpc request, we should use it instead of `req.OsFamily`.

Reply:
Could you explain this a little more?

Reply:
fixed https://github.com/aquasecurity/trivy/pull/433/commits/fc2eed5def774ca6d0a6e3daa59f865d2cfa83c3